tutorials/parmest_demo/parmest_new_API_example.ipynb

```diff
@@ -0,0 +1,942 @@
+    "                         m.b0, m.b1, m.b2, m.b3, m.b4, \n",
+    "                         m.c0, m.c1, m.c2, m.c3, m.c4,\n",
+    "                         m.d0, m.d1, m.d2, m.d3, m.d4,\n",
+    "                         m.e0, m.e1, m.e2, m.e3, m.e4]\n",
+    "        m.unknown_parameters.update(\n",
+    "            (k, pyo.ComponentUID(k)) for k in parameter_set\n",
+    "        )\n",
+    "\n",
+    "        return m\n",
+    "\n",
+    "    def get_labeled_model(self):\n",
+    "        m = self.create_model()\n",
+    "        m = self.finalize_model()\n",
+    "        m = self.label_model()\n",
+    "\n",
+    "        return m"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 5: Solve the parameter estimation problem\n",
+    "\n",
+    "Now, we need to define the following items for parmest to solve the parameter estimation problem: \n",
+    "\n",
+    "    - exp_list\n",
+    "    - objective_function\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Step 5a: Set up the experimental list\n",
+    "\n",
+    "Set up the experimental list by appending the object in the ``Design Experiment`` to a list"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create a experiment list\n",
+    "exp_list = []\n",
+    "for i in range(data_formatted.shape[0]):\n",
+    "    exp_list.append(DesignExperiment(data_formatted, i))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Step 5b: Define an objective function\n",
+    "\n",
+    "Now, we should define an objective function for the parameter estimation. This is the deviation between the observation and the prediction typically chosen to be the sum of squared errors.\n",
+    "\n",
+    "$$\n",
+    "\\sum_{i=0}^n (observation_i - prediction_i)^2 \n",
+    "$$\n",
+    "\n",
+    "We can use the built-in objective function in Parmest to compute the sum of squared errors (``“SSE”``) between the ``m.experiment_outputs`` model values and data values."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Step 5c: Set up the problem\n",
+    "\n",
+    "Set up the parameter estimation problem by creating an instance of the parmest ``Estimator`` object and feed it the required inputs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create an instance of the parmest estimator  \n",
+    "pest = parmest.Estimator(exp_list, obj_function='SSE')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Step 5d: Solve the parameter estimation problem \n",
+    "\n",
+    "Solve the parameter estimation problem by calling theta_est. This will use the entire data set to perform the parameter estimation. \n",
```

Comment:
```suggestion
    "Solve the parameter estimation problem by calling ``theta_est``. This will use the entire data set to perform the parameter estimation. \n",
```